exercises/practice/dnd-character/dnd-character_spec.lua

```diff
@@ -167,13 +162,12 @@ describe('dnd', function()
       assert.equal("integer", math.type(charisma))
       assert.lteq(3, charisma)
       assert.lteq(charisma, 18)
-      assert.equal(charisma, character.charisma)
     end)
 
     it('creates a character with valid hitpoints', function()
       for i = 1, 10 do
         local character = dnd.Character:new(tostring(i))
-        assert.equal(character.hitpoints, 10 + dnd.modifier(character.constitution))
+        assert.equal(10 + dnd.modifier(character.constitution), character.hitpoints)
```

Comment:
The order must be expected, actual in order for the error message to be correct.